cmd/root.go

```diff
@@ -104,7 +106,7 @@ func getConfigDir() string {
 		fmt.Println(err)
 		os.Exit(1)
 	}
-	if _, err := os.Stat(config); os.IsNotExist(err) {
+	if _, err := os.Stat(config); errors.Is(err, fs.ErrNotExist) {
```

Comment:
I've seen this warning in new Golang documentation:
```
This function predates errors.Is. It only supports errors returned by the os package. New code should use errors.Is(err, fs.ErrNotExist). 
```
So, in our case:
- Is there any functional change in behavior? Did you spot a related bug?
- Or, is this just a response to the mentioned Golang warning?

This is just a question to have more context.

Reply:
That last part. I just happened to see the warning and thought as part of trying to fix a bug in #222  I came across this warning. I was trying to apply the [boy scout rule](https://matheus.ro/2017/12/11/clean-code-boy-scout-rule/).